BAZProjectC1/MovieAPI.swift

```diff
@@ -25,7 +25,7 @@ class MovieAPI {
             if let id = result.object(forKey: "id") as? Int,
                let title = result.object(forKey: "title") as? String,
                let poster_path = result.object(forKey: "poster_path") as? String, let strDetail = result.object(forKey: "overview") as? String {
-               movies.append(Movie(id: id, title: title, poster_path: poster_path))
+                movies.append(Movie(id: id, title: title, descriptionMovie: strDetail, posterPath: poster_path))
```

Comment:
Se te fue un espacio y el por favor renombra `poster_path` a `posterPath` para cumplir con la convención.

```suggestion
               movies.append(Movie(id: id, title: title, descriptionMovie: strDetail, posterPath: posterPath))
```